frontend/src/components/Header.tsx

```diff
@@ -5,14 +5,15 @@ export default function Header(){
     const [search, setSearch] = useState<string>("")
     return(
         <header className="Header">
-            <img className="Header-Picture" src={"https://img.freepik.com/vektoren-kostenlos/niedliche-pinguin-spielkarikatur-illustration_138676-2741.jpg?t=st=1653234529~exp=1653235129~hmac=a3e1f0d249c33437e7af45961bb45d52ca59e9deb32b3b23b3efd5d3f91aad17&w=826"} alt="logo"/>
+            <img className="Header-Picture" src={"https://img.freepik.com/vektoren-kostenlos/weinleseabzeichen-der-hand-die-joystickvektorillustration-haelt-rundes-etikett-mit-gamepad_74855-11224.jpg?t=st=1653239105~exp=1653239705~hmac=d536938f4c420a81b8469dbcd43580bb07c01de0a1a5a5e9fc8443c8143489fa&w=826"} alt="logo"/>
             <div className="Search-Bar"><input type={"text"}
```

Comment:
```suggestion
            <div className="Search-Bar">
            <input type={"text"}
```
etwas übersichtlicher, wenn input element in der nächsten Zeile ist